### ISSUE TITLE
[Screen Reader- Adaptive Cards- Import- Product Video] Caption does not define for video.

### ISSUE THREAD

**Issue by MohitTyagi13** [microsoft/AdaptiveCards#7549]
### Target Platforms

iOS, Android

### SDK Version

2021.10.15.1 (40331737)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device:  Samsung Galaxy S21+
Android Version: 12
App: Adaptive Cards
Version: 2021.10.15.1 (40331737)
Screen readers: Talkback

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2.  Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Product Video'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate all the controls under 'Visual'.  
8. Navigate to video and select 'Play button. Navigate all the video controls.
9. Verify the issue.

**Actual Result:**
Caption does not define for video.

**Expected Result:**
Caption should be defined for pre-recorded video.

**Observation:**
**Same issue is also repro for iOS**
Device: iPhone X
iOS Version: 15.5
Application: Adaptive Card
Application Version: 1.0 (2.4.0-beta.20220620.1)

**User Experience: -**
If Caption does not define on video. so, it would be difficult for users to know about video and purpose of video.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.02.02

### Screenshots

https://user-images.githubusercontent.com/95915837/174616558-a894b837-709c-4d67-82ff-d54332c650c0.mp4

https://user-images.githubusercontent.com/95915837/174616741-017d0aef-a218-42ac-a0cd-fea0da414c26.mp4

### Card JSON

```JSON
N/A
```

**Comment by MohitTyagi13** [comment]
https://microsoft.visualstudio.com/OS/_workitems/edit/28383785

**Comment by beervoley** [comment]
@MohitTyagi13 the captions were not implemented yet for IOS and Android. They have only been added for UWP and NodeJS renderers. Please see the status [here](https://github.com/microsoft/AdaptiveCards/issues/4963)

**Comment by vagpt** [comment]
Closing this issue here as this issue is now tracked using URL below.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/46
